src/mozillavpn.cpp

```diff
@@ -401,15 +402,6 @@ void MozillaVPN::maybeStateMain() {
   // getting to the home screen
   if (Feature::get(Feature::Feature_newOnboarding)->isSupported()) {
     if (!settingsHolder->onboardingCompleted()) {
-      // We turn telemetry off when onboarding starts so that the user has to
-      // specifically opt in to all future telemetry *Note: This is needed
-      // because telemetry is on prior to onboarding to record metrics regarding
-      // auth
-      if (!settingsHolder->onboardingStarted()) {
-        settingsHolder->setGleanEnabled(false);
-        settingsHolder->setOnboardingStarted(true);
```

Comment:
What was the line `settingsHolder->setOnboardingStarted(true);` doing before? Wondering if that should still stay?

Reply:
`onboardingStarted` was a way to know that onboarding was shown for the first time (with consecutive shows being possible via quitting and reopening the app). It was meant to disable glean when the onboarding flow is shown for the first time